libobs/obs-scene.c

```diff
@@ -2082,6 +2084,55 @@ void obs_sceneitem_set_order_position(obs_sceneitem_t *item, int position)
 	obs_scene_release(scene);
 }
 
+void obs_scene_set_items_order(obs_scene_t *scene, int64_t* positions, int positions_count)
```

Comment:
Naming is in my opinion poorly chosen here. If my reading of the code is correct, it should be the following:
 - positions: new_items_order
 - positions_count: nb_items